README.md

```diff
@@ -1,2 +1,38 @@
+---
+title: "CFSS hw01"
+author: "Yanning Cui"
+date: "January 9, 2017"
+output: pdf_document
+---
```

Comment:
The table was neat and comprehensive. I think it'd be helpful for other students if you included the pdf output from your code in your repository as well.